scot/datasets.py

```diff
@@ -0,0 +1,85 @@
+# Released under The MIT License (MIT)
+# http://opensource.org/licenses/MIT
+# Copyright (c) 2016 SCoT Development Team
+
+from os import makedirs
+from os.path import expanduser, isfile, isdir, join
+from requests import get
```

Comment:
Is requests a common enough library that depending on it is not too troublesome for potential users?